samples/security/spire/README.md

```diff
@@ -0,0 +1,44 @@
+# Integrating SPIRE through Envoy SDS
+
+There are currently multiple forms of integrating third-party Certificate Authorities (CA) whitin Istio. This example deploys a setup of [SPIRE](https://github.com/spiffe/spire) (the SPIFFE Runtime Environment) as an Exernal CA, through [Envoy's SDS API integration](https://github.com/alexandrealvino/istio.io/blob/6eff1cd45dfce244978cafc10a88f7620c0438fc/content/en/docs/ops/integrations/spire/index.md).
+
+To assist this deployment, two additional components will be shipped together with SPIRE, the [CSI Driver](https://github.com/spiffe/spiffe-csi) and the [Kubernetes Workload Registrar](https://github.com/spiffe/spire/tree/main/support/k8s/k8s-workload-registrar).
+
+The desired behaviour will get triggered during istio-agent bootstrap, in case a socket file under `/var/run/secrets/workload-spiffe-uds/socket` is detected. This tells istio-agent **not** to start its own SDS server, but to use the one provided via socket.
+
+
+## Usage
+
+1. Deploy Spire:
+
+    ```bash
+    kubectl apply -f spire.yaml
```

Comment:
I did take a look at most of the other similar README's under /samples and they all go without the $. Do you have any reference for this?

Reply:
To me the $ helps to show that the blocks with multiple commands are multiple commands and not one long command, like your step 2. Also the Bookinfo README does use $ for commands that users type (though not for command syntax). The main [Istio docs](https://istio.io/latest/docs/examples/bookinfo/#start-the-application-services) include the command prompts as a [standard](https://istio.io/latest/docs/releases/contribute/code-blocks/). But true, most of the other READMEs in /samples don't use the $.

Reply:
Giving a deeper thought, 2 separate commands in the same block looks weird to me without the dollar sign, and because we'll have output being showed in those blocks, we'll need to have that differentiated. Thanks!